src/UI/Components/SplashScreen/StaticScreen/StaticScreen.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { View, StyleSheet, Text, Image, TouchableOpacity, Dimensions, ImageStyle, TextStyle, ViewStyle } from 'react-native';
+import constants from '../../../../UI-Constants/Constant.json'; // Adjusted import path
+
+const { width, height } = Dimensions.get('window');
+
+const StaticScreen: React.FC = () => {
+  return (
+    <View style={styles.container}>
+      <Image
+        source={require("./assets/screenlight.png")}
+        style={styles.image}
+      />
+      <Text style={styles.title}>{constants.TextConstants.TITLE}</Text>
+      <TouchableOpacity style={styles.button}>
+        <Text style={styles.buttonText}>{constants.TextConstants.BUTTON_TEXT}</Text>
+      </TouchableOpacity>
+      <Text style={styles.footerText}>{constants.TextConstants.FOOTER_TEXT}</Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
styles should be in separate file as already told.

Reply:
moved styles in a different file in same folder.